pkgs/development/python-modules/dnspython/default.nix

```diff
@@ -19,11 +21,15 @@ buildPythonPackage rec {
 
   checkInputs = [
     pytestCheckHook
+  ] ++ lib.optional stdenv.isDarwin [
+    cacert
   ];
 
   disabledTests = [
     # dns.exception.SyntaxError: protocol not found
     "test_misc_good_WKS_text"
+    # fails if IPv6 isn't available
+    "test_resolver_override"
```

Comment:
Is this a darwin-only fix? I would have thought the build sandbox would prevent host IPv6 setup from being visible to the build?

Reply:
that sounds like the origin of the error.

Either way, this fixes an existing regression. We can explore alternatives in another PR.